core/consensus/grandpa/impl/grandpa_impl.hpp

```diff
@@ -295,6 +296,8 @@ namespace kagome::consensus::grandpa {
      */
     void loadMissingBlocks(GrandpaContext &&grandpa_context);
 
+    const size_t kVotesCacheSize
```

Comment:
Where is the value? What it equals to? Where is a semicolon at least?

Reply:
Fixed